db/migrate/20220709110201_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :post_id
+      t.integer :author_id
```

Comment:
- [ ] Kindly apply apply the previous comment given on the likes migration on the comments